drivers/include/periph/comms_bus.h

```diff
@@ -0,0 +1,136 @@
+/*
+ * Copyright (C) 2020 Pieter du Preez <pdupreez@gmail.com>
+ *               2018 Freie Universität Berlin
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @defgroup    drivers_periph_comms_bus (I2C/SPI)
+ * @ingroup     drivers_periph
+ * @brief       Low-level I2C/SPI bus driver interface
+ *
+ * This is a kind of router for drivers that are able to support
+ * either I2C or API bus interfaces.
+ *
+ * It is required to call the comms_bus_setup function as start-up.
+ * This function sets up function pointers for directing bus
+ * communication to the desired bus.
+ *
+ * @{
+ *
+ * @file
+ * @brief       Low-level I2C/SPI bus driver interface definition
+ *
+ * @author      Pieter du Preez <pdupreez@gmail.com>
+ */
+
+#ifndef PERIPH_COMMS_BUS_H
+#define PERIPH_COMMS_BUS_H
+
+#if !defined(MODULE_PERIPH_I2C) && !defined(MODULE_PERIPH_SPI)
```

Comment:
This check breaks the CI build for almost all boards.
One option for fixing it, could be to simply remove this check.
I'm not sure what the best way would be to fix this?